### PR TITLE
Redirect to homepage if not set session name

### PR DESCRIPTION
**Pull request by hiveer**

### CODE REVIEW

**Comment by just4fun** [comment]
We should also check whether user has inputed session name and user name, or we can click buttons directly to session page as well.

File: scripts/session.js [hunk 1]
```diff
@@ -5,6 +5,11 @@ var sessionName = Cookies.get("session_name");
 var userName = Cookies.get("user_name");
 var cookie_point = Cookies.get("point");
 
+// redirect to login page
+if(sessionName == undefined) {
```
Comment (just4fun) [inline scripts/session.js]: `if (!sessionName)` is enough.

In addition, I think we should check both user name and session name here.

**Comment by just4fun** [comment]
Regarding `join_session` page, we should also check user name, or we can access session page with session name only.

File: scripts/session.js (hunk 1 above)
Comment (hiveer) [inline scripts/session.js]: @just4fun  done

**Comment by just4fun** [comment]
Looks good to me now.

We can add check for input fields once we update UI in near future, now the PR is used to avoid user to join session if there is no user name or session name.

**Comment by just4fun** [comment]
giphy

**Pull request merged**